src/QuestionDrawer.js

```diff
@@ -5,27 +5,24 @@ import ReactSlider from 'react-slider'
 
 import ButtonSurvey from './ButtonSurvey'
 
-
 // import ReactSliderAge from './ReactSliderAge'
 import CancerTypes from './data/CancerTypes'
 
-
 class QuestionDrawer extends React.Component {
     componentWillMount () {
```

Comment:
Only 1 `componenWillMount` and `componentWillReceiveProps` function is allowed per component; the `setState` function should set both variables at once. Because there were 2 such functions before, only 1 would "win" and get set.